src/cmds/build/index.ts

```diff
@@ -1,17 +1,15 @@
-const builders = require('../../builders');
-const {
-  getOutfilePath,
-  validateDirPath,
-  validateFilePath,
-  validateOutfileName,
-} = require('../../utils');
-const { handler: snapEval } = require('../eval');
-const { handler: manifest } = require('../manifest');
-const { bundle } = require('./bundle');
+import builders from '../../builders';
+import { getOutfilePath, validateDirPath, validateFilePath, validateOutfileName } from '../../utils';
+import { snapEval } from '../eval';
+import { manifest } from '../manifest/manifest';
+import { bundle } from './bundle';
+
+/* Custom Type Imports */
+import { Argument } from "../../types/yargs";
```

Comment:
There's no need to separate these by a newline, and the comment should of course get tossed out.